romana-install/roles/romana/postinstall/kubernetes/master/templates/demo.sh

```diff
@@ -94,10 +94,10 @@ desc "let's try to have the frontend load data from the backend"
 run "kubectl --namespace=tenant-a exec nginx-frontend -- curl $(get_pod_ip 'nginx-backend' 'tenant-a') --connect-timeout 5"
 
 desc "now let's add a policy that permits frontend to connect to the backend"
-run "curl -X POST -H "Content-ype: application/yaml" -d @romana-np-frontend-to-backend.json http://192.168.99.10:8080/apis/romana.io/demo/v1/namespaces/tenant-a/networkpolicys; sleep 5"
+run "curl -X POST -H 'Content-ype: application/yaml' -d @romana-np-frontend-to-backend.json http://{{ romana_master_ip }}:8080/apis/romana.io/demo/v1/namespaces/tenant-a/networkpolicys; sleep 5"
```

Comment:
yep just realized that :) previously it was ignore because of quotes, now it fails there, fixed it now.